src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -680,10 +680,35 @@ bool SQLPTRepresentation::GatherConsumerFriendlyMessages(
   if (query.Prepare(sql_pt::kCollectFriendlyMsg)) {
     while (query.Next()) {
       UserFriendlyMessage msg;
+
+      msg.tts = query.GetString(1);
+      msg.label = query.GetString(2);
+      msg.line1 = query.GetString(3);
+      msg.line2 = query.GetString(4);
+      msg.text_body = query.GetString(5);
       msg.message_code = query.GetString(7);
-      std::string language = query.GetString(6);
 
-      (*messages->messages)[msg.message_code].languages[language];
+      std::string language = query.GetString(6);
+      if (!msg.tts.empty()) {
+        *(*messages->messages)[msg.message_code].languages[language].tts =
```

Comment:
@jacobkeeler This dereferencing looks a bit confusing. Is it possible to create a reference to ` *(*messages->messages)[msg.message_code].languages` and use it like: 
```cpp
languages[language].tts = msg.tts 
```  

or even better to create reference to the entire expression `*(*messages->messages)[msg.message_code].languages[language]`
and use it like: 

```cpp
languages.tts = msg.tts
languages.label = msg.label

```

Reply:
@AByzhynar This was copied from the EXTERNAL_PROPRIETARY policy code, so I will make this change there as well.